src/app/x/index.ts

```diff
@@ -61,16 +68,22 @@ export const services = (app: Record<string, Token>): ServiceDefinition[] => {
     [$.xVueComponents, {
       service: () => {
         return [
+          ['XCard', KCard],
+          ['XPop', KPop],
+          ['XInputSwitch', KInputSwitch],
+          ['XCheckbox', KCheckbox],
```

Comment:
This is how we rename this right?

Reply:
Yup, for a rename with no extra functionality this says "when I write <XCard>, use KCard".

Then if we ever need to wrap KCard with extra functionality we just make a XCard that wraps KCard and we change this to `['XCard', XCard]`.

You'll see we've already needed to do this wrapping with `['XBreadcrumbs', XBreadcrumbs]` for example (inside `XBreadcrumbs` we use `KBreadcrumbs` but add extra functionality).

Currently its confusing having two prefixes (K and X) depending on whether we need to add functionality or not, so we are in the process of just renaming them all to X even if they don't currently ned wrapping.